src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/config/work-experience-reference-decline-form.ts

```diff
@@ -7,8 +7,20 @@ const workExperienceDeclineForm: Form = {
   inputs: {
     referenceDecline: {
       id: "decline",
-      component: EceTextField,
-      props: { label: "user declined this does not work" },
+      component: EceRadio,
+      props: {
+        options: [
+          { key: "Iamunabletoatthistime", label: "I am unable to at this time" },
```

Comment:
**Nitpick** but can we please case these as such: `iAmUnableToAtThisTime`...

Reply:
This comes from backend and dynamics, should be match to Enum items I think